File: Colleen/CPP/Colleen.cpp
#include <iostream>
#include <string>
#include <algorithm>

std::string s = R"(#include <iostream>
#include <string>
#include <algorithm>

std::string s = R*($)*;

// Global Scope Comment
int main() {
    // Main Scope Comment
    std::string c = s;
    std::replace(s.begin(), s.begin() + s.find('$') + 5, '*', '"');
    std::cout << s.substr(0, s.find('$')) << c << s.substr(s.find('$') + 1);
}
)";

// Global Scope Comment
int main() {
    // Main Scope Comment
    std::string c = s;
    std::replace(s.begin(), s.begin() + s.find('$') + 5, '*', '"');
    std::cout << s.substr(0, s.find('$')) << c << s.substr(s.find('$') + 1);
}

File: Colleen/Java/Colleen.java
// Global Scope Comment
public class Colleen {
  public static void main(String[] args) {
    // Main Scope Comment
    char q = 34;
    String[] l = {
    "// Global Scope Comment",
    "public class Colleen {",
    "  public static void main(String[] args) {",
    "    // Main Scope Comment",
    "    char q = 34;",
    "    String[] l = {",
    "    ",
    "    };",
    "    for(int i = 0; i < 6; i++)",
    "      System.out.println(l[i]);",
    "    for(int i = 0; i < l.length; i++)",
    "      System.out.println(l[6] + q + l[i] + q + ',');",
    "    for(int i = 7; i < l.length; i++)",
    "      System.out.println(l[i]);",
    "  }",
    "}",
    };
    for(int i = 0; i < 6; i++)
      System.out.println(l[i]);
    for(int i = 0; i < l.length; i++)
      System.out.println(l[6] + q + l[i] + q + ',');
    for(int i = 7; i < l.length; i++)
      System.out.println(l[i]);
  }
}
